test/modules/layers/test_quantization.py

```diff
@@ -38,11 +38,14 @@ def setUp(self):
         )
 
         self.vq = Quantization(
-            num_embeddings=self.num_embeddings, embedding_dim=self.embedding_dim
+            num_embeddings=self.num_embeddings,
+            embedding_dim=self.embedding_dim,
+            decay=0.5,
         )
-        self.vq.embedding = nn.Embedding.from_pretrained(self.embedding_weights)
 
     def test_quantized_output(self):
+        self.vq.embedding = self.embedding_weights
+        self.vq._is_embedding_init = True
```

Comment:
nit: does _is_embedding_init need to be private? In general I would try to not be setting variables that we claim are private like this. Obviously it's not really private but the intention is implied by the leading underscore (which is why this is only a nit)

Reply:
for the purposes of the test I had to set a private variable, but I would imagine for general use I wouldn't want this variable to be modified without knowledgeable intent